tests/test_phishingpage.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python2
+
+import unittest
+import phishingpage
+import os
+
+
+class TestUrlChecker(unittest.TestCase):
```

Comment:
Most of the tests here are for urllib module. We don't have to include them. Your tests should include only the logic that you inserted. Having said that, I believe for the url checking you need only one test that recognizes false urls and one that recognizes 404 pages.